packages/preview-middleware-client/test/unit/adp/init.test.ts

```diff
@@ -99,8 +99,41 @@ describe('adp', () => {
 
         expect(sendActionMock).toHaveBeenNthCalledWith(2, {
             type: '[ext] show-dialog-message',
-            payload:
-                'The current SAPUI5 version set for this Adaptation project is 1.70.0. The minimum version to use for SAPUI5 Adaptation Project and its SAPUI5 Visual Editor is 1.71'
+            payload: {
+                message:
+                    'The current SAPUI5 version set for this Adaptation project is 1.70.0. The minimum version to use for SAPUI5 Adaptation Project and its SAPUI5 Visual Editor is 1.71',
+                shouldHideIframe: true
+            }
+        });
+    });
+
+    test('init - send notification existance of sync views', async () => {
```

Comment:
Spelling - exist**e**nce
```suggestion
    test('init - send notification existence of sync views', async () => {
```

Reply:
Done